contrib/apiage.py

```diff
@@ -259,7 +259,11 @@ def format_markdown(tracked, outfh):
             print("", file=outfh)
 
 
-def format_updates_markdown(updates, outfh):
+def format_updates_markdown(updates, outfh, issuetemplate=False, next_ver=""):
+    if issuetemplate:
+        print("---", file=outfh)
+        print(f"title: APIs pending stability updates in {next_ver}", file=outfh)
```

Comment:
Reading through [create-an-issue docs](https://github.com/JasonEtco/create-an-issue#usage) I came to know about an option to specify labels too.
```
print(f"labels: no-API", file=outfh)
```

Reply:
I don't think any labels  are needed for this since its an Issue not a PR.